docs/menu.yml

```diff
@@ -188,6 +188,10 @@ catalog:
                 path: "/en/setup/service-agent/virtual-cache"
               - name: "Virtual MQ"
                 path: "/en/setup/service-agent/virtual-mq"
+          - name: "Workflow Scheduler"
```

Comment:
<img width="373" alt="image" src="https://github.com/apache/skywalking/assets/5441976/b0899a67-3425-4457-9db0-c458d13831a2">

You should put the menu item here. Rather than randomly adding into the first one.